sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -0,0 +1,853 @@
+                    request_body, api_version=constants.API_VERSION, **kwargs)
+            )
+
+        entry = QueueDescriptionEntry.deserialize(entry_ele)
+        result = QueueDescription._from_internal_entity(
+            entry.content.queue_description)
+        result.name = queue_name
+        return result
+
+    async def update_queue(self, queue_description: QueueDescription, **kwargs) -> None:
+        """Update a queue.
+
+        :param queue_description: The properties of this `QueueDescription` will be applied to the queue in
+         ServiceBus. Only a portion of properties can be updated.
+         Refer to https://docs.microsoft.com/en-us/rest/api/servicebus/update-queue.
+        :type queue_description: ~azure.servicebus.management.QueueDescription
+        :keyword timedelta default_message_time_to_live: The value you want to update to.
+        :keyword timedelta lock_duration: The value you want to update to.
+        :keyword bool dead_lettering_on_message_expiration: The value you want to update to.
+        :keyword timedelta duplicate_detection_history_time_window: The value you want to update to.
+        :keyword int max_delivery_count: The value you want to update to.
+        :rtype: ~azure.servicebus.management.QueueDescription
+        """
+
+        if not isinstance(queue_description, QueueDescription):
+            raise TypeError("queue_description must be of type QueueDescription")
+
+        internal_description = queue_description._to_internal_entity()
+        to_update = copy(internal_description)
+
+        to_update.default_message_time_to_live = kwargs.get(
+            "default_message_time_to_live") or queue_description.default_message_time_to_live
+        to_update.lock_duration = kwargs.get("lock_duration") or queue_description.lock_duration
+        to_update.dead_lettering_on_message_expiration = kwargs.get(
+            "dead_lettering_on_message_expiration") or queue_description.dead_lettering_on_message_expiration
+        to_update.duplicate_detection_history_time_window = kwargs.get(
+            "duplicate_detection_history_time_window") or queue_description.duplicate_detection_history_time_window
+        to_update.max_delivery_count = kwargs.get("max_delivery_count") or queue_description.max_delivery_count
+
+        to_update.default_message_time_to_live = avoid_timedelta_overflow(to_update.default_message_time_to_live)
+        to_update.auto_delete_on_idle = avoid_timedelta_overflow(to_update.auto_delete_on_idle)
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_update,
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        with _handle_response_error():
+            await self._impl.entity.put(
+                queue_description.name,  # type: ignore
+                request_body,
+                api_version=constants.API_VERSION,
```

Comment:
Is there a reason we're not passing API_VERSION elsewhere?

Reply:
"elsewhere" means other methods of this client, or other SDKs?

Reply:
I think we're passing it in every http operation -- get/put/delete

Reply:
Yes I had recalled endpoints where I didn't see this but if I'm mistake don't hesitate to ignore this.  I need to take better notes for myself as I read big PRs.